apps/tailwind-components/pages/Message.story.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <Message type="error">Some error</Message>
```

Comment:
The invalid styles do not show in the preview or locally

Reply:
fixed